scripts/generate-openapi.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -ux
+SCRIPT_DIR="$(
+  cd -- "$(dirname "$0")" >/dev/null 2>&1 || exit
+  pwd -P
+)"
+PROJECT_ROOT="${SCRIPT_DIR}"/..
+SWAGGER_SPEC_FILE="${PROJECT_ROOT}/transcendence-app/swagger-spec.yaml"
+mkdir -p "${PROJECT_ROOT}"/webapp/src/shared/generated
+ME="$(whoami)"
+DOCKER_COMPOSE="$("${PROJECT_ROOT}"/scripts/get-docker-compose.sh)"
+cd "${PROJECT_ROOT}" && touch "${SWAGGER_SPEC_FILE}"
+{ { cd "${SCRIPT_DIR}" || exit ; } &&
+${DOCKER_COMPOSE} run --rm swagger && ${DOCKER_COMPOSE} down -v ; } ||
+  rm "${SWAGGER_SPEC_FILE}"
+docker run --rm -e MY_USER="${ME}" \
```

Comment:
Maybe we would like this part of the script in a way that it can be ran independently from the generation of the yaml file, to execute manually when hot reload updates the source

Reply:
I think is ok as is. If we need to reload the autogenerated interfaces `make gen` seems enough

Reply:
I still added functionality to the script to ba able to run either separately.